src/js/hca/components/HealthCareApp.jsx

```diff
@@ -129,23 +131,38 @@ class HealthCareApp extends React.Component {
     e.preventDefault();
     const veteran = this.props.data;
     const path = this.props.location.pathname;
+    let apiUrl = `${window.VetsGov.api.url}/api/hca/v1/application`;
+    let formSubmissionId;
+    let timestamp;
+    const testBuild = __BUILDTYPE__ === 'development' || __BUILDTYPE__ === 'staging';
+    const submissionPost = {
+      method: 'POST',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json'
+      },
+      timeout: 10000, // 10 seconds
+      body: veteranToApplication(veteran)
+    };
 
     window.dataLayer.push({ event: 'submit-button-clicked' });
     const formIsValid = validations.isValidForm(veteran);
 
+    if (testBuild) {
```

Comment:
Will add comment, thanks! And good to know, since this is temporary I may not change things now but will definitely take advantage of that in the future, this was pretty complicated to hack together :)